README.md

```diff
@@ -1,12 +1,15 @@
+[![Maven Central Version](https://img.shields.io/maven-central/v/com.github.nradov/abnffuzzer)](https://central.sonatype.com/artifact/com.github.nradov/abnffuzzer)
```

Comment:
I hope this `com.github.nradov:abnffuzzer` artifact is actually what you published.